models/agent_helpers.go

```diff
@@ -827,6 +828,7 @@ func CreateAgent(q *reform.Querier, agentType AgentType, params *CreateAgentPara
 		AzureOptions:                   params.AzureOptions,
 		PushMetrics:                    params.PushMetrics,
 		DisabledCollectors:             params.DisableCollectors,
+		LogLevel:                       &params.LogLevel,
```

Comment:
it will make LogLevel allways not nil. let's use `pointer.ToStringOrNil`

Reply:
fix